optax/_src/utils.py

```diff
@@ -25,12 +25,12 @@
 import jax
 import jax.numpy as jnp
 import jax.scipy.stats.norm as multivariate_normal
+from jax import tree_util
```

Comment:
jax prefers the full `jax.tree.map` in place of `tree_util.map` - could you please update uses of `tree_util` and remove the import